ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -263,7 +256,7 @@ export class Auth0AdapterService implements OnDestroy {
 
     public logout(returnToUrl: string = ''): void {
         const baseUrl = this.configuration.baseUrl;
-        const wasAdmin = this.session.session.isAdmin;
+        const wasAdmin = this.session.session && this.session.session.isAdmin;
```

Comment:
We might not have a session object if we failed login, so adding a safeguard here.